docs/code/README.md

```diff
@@ -0,0 +1,29 @@
+# Code
+
+## Architecture
```

Comment:
I don't agree that the most important part of our architecture is our module system. I think it is worth mentioning the adapter/foundation pattern as the foundation (heh) of our architecture, then end with how we define subsystems, components, and modules.